src/client/components/CircleMakerComponent/MapCircleMarker.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import L from 'leaflet';
+import { Map, TileLayer, Marker, Popup, CircleMarker } from 'react-leaflet';
+import 'leaflet/dist/leaflet.css';
+import './MapCircleMarker.css';
+import iconSrc from '../../assets/images/icons/pin-icon.svg';
+
+const markerIcon = L.icon({
+  iconUrl: iconSrc,
+  iconAnchor: [12.5, 41],
+  iconSize: [35, 41],
+  popupAnchor: [0, -41],
+});
+
+class MapCircleMarker extends Component {
```

Comment:
Can you rewrite this as a function instead? This is what we have been using so far